app/Http/Controllers/BookingRequestController.php

```diff
@@ -173,10 +186,18 @@ public function edit(BookingRequest $booking)
      */
     public function update(UpdateBookingRequest $request, BookingRequest $booking)
     {
+
+        //If comments don't call this
+        if ($booking->status == BookingRequest::REVIEW || $booking->status == BookingRequest::APPROVED ) {
+            return redirect()->route('bookings.view', ['booking' => $booking]);
```

Comment:
Could we flash a message to explain the redirect to the user plz

Reply:
ditto